server/routes/api/place.py

```diff
@@ -380,22 +384,60 @@ def get_related_place(dcid,
                                 is_per_capita=is_per_capita)
 
 
-@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@cache.memoize(timeout=3600 * 24)
+def get_place_cohort(cohort):
+    """Get all the place dcids for a cohort set."""
+    return dc.get_property_values([cohort], 'member')[cohort]
+
+
 @bp.route('/similar/<stats_var>/<path:dcid>')
 def api_similar_places(stats_var, dcid):
     """
-    Get the similar places for a given place by stats var within the same place.
+    Get the similar places for a given place.
+
+    If the place is a USA city or county, the similar places are randomly
+    selected from curated cohort. Otherwise, the similar places are by stats
+    var within the same place.
     """
+    # Choose city from US city cohort.
+    match = re.match(r'geoId/\d{7}', dcid)
+    if match:
+        city_cohort = get_place_cohort(CITY_COHORT)
+        random.shuffle(city_cohort)
```

Comment:
let's seed this so the results are somewhat stable, maybe with the dcid?

Reply:
Added a seed with current day of the year. If with dcid then would not have the random list effect for a place..

Reply:
Actually dcid + year of the day maybe better since each place would see different places?

Reply:
ok lets try this.